app/retail/templates/shared/analytics.html

```diff
@@ -19,15 +19,19 @@
   <script async>(function(s,u,m,o,j,v){j=u.createElement(m);v=u.getElementsByTagName(m)[0];j.async=1;j.src=o;j.dataset.sumoSiteId='0f081a6f95a3644ff88bf8da28b0ac85e88083ae84b6d685a8039be24680e53c';v.parentNode.insertBefore(j,v)})(window,document,'script','//load.sumo.com/');</script>
   {% endif %}
 {% endif %}
-{% if env == 'prod' %}
+{% if env == 'prod' or 1 %}
```

Comment:
or 1 ?

Reply:
this is just me testing.  should remove before merge